raft/rafttest/interaction_env_handler_add_nodes.go

```diff
@@ -43,10 +44,12 @@ func (env *InteractionEnv) handleAddNodes(t *testing.T, d datadriven.TestData) e
 				arg.Scan(t, i, &snap.Metadata.Index)
 			case "content":
 				arg.Scan(t, i, &snap.Data)
+			case "async-storage-writes":
+				arg.Scan(t, i, &asyncStorageWrites)
```

Comment:
Consider reading into a `Config` directly, like in ffee6cc4bd03f1ab8c7f328ee6b11a13b1c2ed49.

Reply:
Nice, I like this. Once https://github.com/etcd-io/etcd/pull/14633 lands, I'll rebase on top of it and make this change.

Reply:
Done.